### ISSUE TITLE
no uvx.exe found on windows during installation

### ISSUE THREAD

**Issue by njzjz** [astral-sh/uv#4755]
```
downloading uv 0.2.20 x86_64-pc-windows-gnu
installing to /c/Users/runneradmin/.cargo/bin
  uv.exe
mv: cannot stat '/tmp/tmp.NnFUbsjG4h/uvx.exe': No such file or directory
ERROR: command failed: mv /tmp/tmp.NnFUbsjG4h/uvx.exe /c/Users/runneradmin/.cargo/bin
Error: Process completed with exit code 1.
```

A simple GitHub Action can reproduce it:

```yml
name: Test

on:
  push:

jobs:
  test:
    name: test windows uv
    runs-on: windows-latest
    steps:
    - run: curl -LsSf https://astral.sh/uv/install.sh | sh
      shell: sh
```

When using the powershell instead of shell, it still cannot find `uvx.exe`, but prints `everything's installed!` and does not fail the script (is it expected?).

```
Run powershell -c "irm https://astral.sh/uv/install.ps1 | iex"
  powershell -c "irm https://astral.sh/uv/install.ps1 | iex"
  shell: C:\Program Files\PowerShell\7\pwsh.EXE -command ". '{0}'"
Downloading uv 0.[2](https://github.com/njzjz/gist/actions/runs/9771365816/job/26974013770#step:2:2).20 (x86_64-pc-windows-msvc)
Installing to C:\Users\runneradmin\.cargo\bin
  uv.exe
Copy-Item : Cannot find path 'C:\Users\runneradmin\AppData\Local\Temp\a68bd8a8-a5da-48ff-8d6[3](https://github.com/njzjz/gist/actions/runs/9771365816/job/26974013770#step:2:3)-5151c2b99750\uvx.exe' 
because it does not exist.
At line:244 char:5
+     Copy-Item "$bin_path" -Destination "$dest_dir"
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\runner...2b99750\uvx.exe:String) [Copy-Item], ItemNotFoundExce 
   ption
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.CopyItemCommand
 
Remove-Item : Cannot find path 'C:\Users\runneradmin\AppData\Local\Temp\a68bd8a8-a5da-[4](https://github.com/njzjz/gist/actions/runs/9771365816/job/26974013770#step:2:5)8ff-8d63-5151c2b99750\uvx.exe' 
because it does not exist.
At line:24[5](https://github.com/njzjz/gist/actions/runs/9771365816/job/26974013770#step:2:6) char:5
+     Remove-Item "$bin_path" -Recurse -Force
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\runner...2b99[7](https://github.com/njzjz/gist/actions/runs/9771365816/job/26974013770#step:2:8)50\uvx.exe:String) [Remove-Item], ItemNotFoundEx 
   ception
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.RemoveItemCommand
 
  uvx.exe
everything's installed!
```

```yml
name: Test

on:
  push:

jobs:
  test:
    name: test windows uv
    runs-on: windows-latest
    steps:
    - run: powershell -c "irm https://astral.sh/uv/install.ps1 | iex"
```

**Comment by zanieb** [comment]
Thanks for the report. I'd recommend pinning to 0.2.18 in the meantime: 

```
powershell -c "irm https://astral.sh/uv/0.2.18/install.ps1 | iex"
```

We'll resolve this ASAP though.

**Comment by zanieb** [comment]
This should be fixed in 0.2.21
